docs/core/versions/lts-current.md

```diff
@@ -29,7 +29,10 @@ Current releases are
 * And one year after the general availability of a subsequent LTS release
 
 ## Versioning
-New LTS releases are marked by an increase in the Major version number. Current releases have the same Major number as the corresponding LTS train and are marked by an increase in the Minor version number. Bug fix updates to either train increment the Patch version. For more information on the versioning scheme, see [.NET Core Versioning](index.md).
+New LTS releases are marked by an increase in the Major version number. Current releases have the same Major number as the corresponding LTS train and are marked by an increase in the Minor version number. For example, 1.0.3 would be LTS and 1.1.0 would be Current. Bug fix updates to either train increment the Patch version. For more information on the versioning scheme, see [.NET Core Versioning](index.md).
+
+## What causes updates in LTS and Current trains?
+To understand what specific changes, such as bug fixes or the addition of APIs, cause updates to the version numbers review the Decision Tree section in the [Versioning Documentation](index.md). There is not a golden set of rules that decide what changes are pulled into the LTS branch from Current. Typically, necessary security updates and patches that fix expected behaviour are reasons to update the LTS branch. We also intend to support recent desktop developer OSes on the LTS branch, though this may not always be possible. A good way to catch up on what APIs, fixes, and OSes are supported in a certain release is to browse its [release notes](https://github.com/dotnet/core/tree/master/release-notes) on GitHub.
```

Comment:
nit: I always see people using the OSes and I finally went to check the style guide and the correct acronym is actually OSs (the rule is you just add an s at the end even if the acronym finishes in S). It's weird so you might want to spell out.

Some entertaining articles on the subject that @Rick-Anderson has found:
http://technotes.whw1.com/computer-related/operating-systems/22-what-is-the-plural-of-os-operating-system
http://itknowledgeexchange.techtarget.com/writing-for-business/the-plural-of-os/